### PR TITLE
CA-151 Allow application subnet to reach logs interface endpoint

### PR DESCRIPTION
**Pull request by fvazquez-caylent**
* Adds application subnet cidr block (Tamr VM subnet cidr block) as allowed cidr block for ingress traffic in the security group for the Cloudwatch Interface Endpoint.

### CODE REVIEW

File: vpc-endpoints.tf [hunk 1]
```diff
@@ -54,7 +54,7 @@ resource "aws_security_group" "logs_interface_endpoint" {
     from_port   = 443
     to_port     = 443
     protocol    = "TCP"
-    cidr_blocks = [aws_subnet.compute_subnet.cidr_block]
+    cidr_blocks = [aws_subnet.compute_subnet.cidr_block, module.vpc.private_subnets_cidr_blocks[0]]
```
Comment (souza-dan) [inline vpc-endpoints.tf:57]: Why is this only the first CIDR block from the VPC's private subnets?

Is that guaranteed to map to where a service would be deployed that would need to use cloudwatch?

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.1.0
+1.2.0
```
Comment (souza-dan) [inline VERSION:1]: Arguably, this may be a patch, since this seems more like a bug fix than a feature.

File: vpc-endpoints.tf (hunk 1 above)
Comment (fvazquez-caylent) [reply vpc-endpoints.tf]: Fixed.

**Pull request merged**